Editor/Document.cs

```diff
@@ -2151,13 +2151,13 @@ public void SerializeAnimationSamplerBundle(Root root, KeyframeAccessorBundle bu
                         ? $"{name.Value}_{indexBufferViewSuffix}"
                         : indexBufferViewSuffix),
                 };
-                var componentType = builder.BaseAccessorCount switch
+                var (indices, values) = builder.Build();
+                var componentType = indices.Max() switch
```

Comment:
[nits]
`indices` は事前に昇順ソートされていることがわかっているので、Last がよいかなと思いました

```suggestion
                var componentType = indices.Last() switch
```

Reply:
確かにその通りですね。
取り込みます。